### ISSUE TITLE
Embedded YouTube video uses HTML to format content

### ISSUE THREAD

**Issue by sbaker-ymcashr** [ymcatwincities/openy#2049]
HTML code is being used to format content (Level A accessibility issue) for all of my embedded YouTube videos. The rendered HTML from embedding the YouTube video through the Drupal embed entity tool has the following attributes: <iframe width="854" height="480" frameborder="0" allowfullscreen="allowfullscreen" src="YOUTUBE VIDEO LINK">

**Comment by podarok** [comment]
@sbaker-ymcashr could you clarify what is the issue here? Is this a duplication of https://github.com/ymcatwincities/openy/issues/2048 ?

**Comment by sbaker-ymcashr** [comment]
Hi @podarok, this is related to #2048 but flagging as a separate issue in SiteImprove.

I'm not 100% certain which tag is being considered formatting, but I am thinking that could be the "frameborder=0" tag, but maybe also the width and height tags as well? Here is the help text from SiteImprove regarding this issue: 

**Description of this issue:**
Presentational attributes such as 'border', 'align', or 'bgcolor' are used. CSS should be used for styling instead.

**How to fix it:**
It's important to review these issues and determine if they can be fixed by moving the presentational attributes to your CSS. It's not always practical to make these changes, but should be made when possible.

They link to this guide: https://www.w3.org/TR/WCAG20-TECHS/G140.html

**Comment by podarok** [comment]
Could be fixed in latest entity_embed which should land in OpenY in Drupal 9 https://github.com/ymcatwincities/openy/pull/2222

**Comment by sarah-halby** [comment]
This will be added to point release 2.9.1.
